.travis.yml

```diff
@@ -14,6 +14,7 @@ language: shell
       echo \"Build triggered by ${TRAVIS_EVENT_TYPE}\"
       export PS4='+ \e[33;1m(\$0 @ line \$LINENO) \$\e[0m '
       set -ex  # -e = exit on failure; -x = trace for debug
+      opam repo -a --set-default add coq-extra-dev https://coq.inria.fr/opam/extra-dev
```

Comment:
As far as I am concerned, this change breaks CI for me. The consequence of this change is that the CI no longer checks compatibility with any released version of `mathcomp`. See, for instance, this [job](https://travis-ci.com/github/coq-community/reglang/jobs/340987214) where even for coq-8.10.x the CI installs `coq-mathcomp-ssreflect.dev`. 

The previous behavior, where released versions of `coq` are tested with the latest stable release of `mathcomp` and `coq.dev` with `mathcomp.dev` was just fine.

Reply:
Thanks for the feedback. That's an interesting consequence. cc @erikmd @liyishuai 
I'll simply revert this change so that you can still merge the PR.

Reply:
A quick solution is adding a boolean flag to control extra-dev in CI.

Reply:
I guess this should be an issue in the `templates` repository, right?

Reply:
Yep. (Leaving this unresolved so that the discussion is still visible.)